### PR TITLE
Add support for multiple KeyVaults

### PR DESCRIPTION
**Pull request by mkah91**
@fbufler it's still very much work in progress, but I wanted to see what you think from a users point of view before continuing.

Fixes: https://github.com/mkah91/azure-keyvault-cli/issues/8

### CODE REVIEW

**Comment by fbufler** [comment]
Looks pretty cool to me.
Some minor things I've found that disturbed my experience a bit.

`azkv vaults list` let's you select vaults (not sure if thats intended by list, maybe we shoud use a set commands idk.

Scrolling through `azkv show` or `azkv edit` is a bit unstructured (search obviously helps a lot)
- Maybe we could only show the vault name if there are duplicate secret names?
- Or we put the vault name in front with `vault/secret` so it's a bit more structured.

**Comment by mkah91** [comment]
Sounds good. If I find some time this week I will come up with an adapted Version (probably still wip, though)

**Comment by mkah91** [comment]
@fbufler I added your suggestions and finished the refactoring (tests are missing), If you find some time, please have a look :)

**Comment by fbufler** [comment]
> @fbufler I added your suggestions and finished the refactoring (tests are missing), If you find some time, please have a look :)

I'm loving it, awesome work :)

**Comment by mkah91** [comment]
@fbufler I finally found some time to at least implement some unit tests. To ship the functionality I'd say it's okay to merge now and increase unit test coverage later. If you find some time please have a look and try it out :)

File: cli/commands/vaults/common.py [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (fbufler) [inline cli/commands/vaults/common.py:1]: I guess this file is not needed or?

**Review by fbufler — APPROVED** [review]
LGMT :) Awesome work btw

**Pull request merged**